app/controllers/webhooks/webhooks.controller.js

```diff
@@ -17,7 +17,7 @@ async function webhookDetailPage (req, res, next) {
   const page = req.query.page || 1
 
   try {
-    const webhook = await webhooksService.getWebhook(req.params.webhookId, req.service.externalId)
+    const webhook = await webhooksService.getWebhook(req.params.webhookId, req.service.externalId, req.account.external_id)
```

Comment:
This would ideally be what all of the different services would use but unfortunately they talk in internal account identifiers. Everywhere it's specified here should be `req.account.gateway_account_id`